beacon_node/lighthouse_network/src/rpc/protocol.rs

```diff
@@ -181,6 +200,9 @@ fn rpc_light_client_finality_update_limits_by_fork(current_fork: ForkName) -> Rp
         ForkName::Deneb => {
             RpcLimits::new(altair_fixed_len, *LIGHT_CLIENT_FINALITY_UPDATE_DENEB_MAX)
         }
+        ForkName::Electra => {
+            unimplemented!("Electra not implemented")
```

Comment:
Why not default to Deneb for now?

Reply:
I'm not familiar with the Electra changes to light clients, and I'd rather not forget to update this when Electra comes. What's the value of adding a value that might be obsolete? We're not going to hit this code path pre Electra.

Reply:
I worry we forget, enable the light-client server by default and LH gets panics on devnets / testnets. Could add a default to electra and a `TODO(electra)`

Reply:
Done in 51fe0ce83cb3eeabee4e367424bb9aa20e91537c